### ISSUE TITLE
Add config files for editing machine recipes

### ISSUE THREAD

**Issue by CapitalismWouldveWorked** [TheOriginalGolem/Hbm-s-Nuclear-Tech-GIT#63]
We already have a config file dedicated to the assembler, it would be great to have one for other machines like the shredder or chemical plant. Though I'm not sure how hard this may be. Thanks!

**Comment by Copernicium282** [comment]
Yeah that would be way better for modpack devs to integrate the mod with other mods, and i had been waiting for such a thing after you did the config for he to rf for me
